taxcalc/functions.py

```diff
@@ -1059,27 +1079,25 @@ def F2441(_earned, _fixeic, e59560, MARS, f2441, DCC_c,
 
 
 @iterate_jit(nopython=True)
-def DepCareBen(c32800, _cmp, MARS, c32880, c32890, e33420, e33430, e33450,
-               e33460, e33465, e33470, _sep, _dclim, e32750, e32775,
+def DepCareBen(c32800, _cmp, f2441, MARS, c32880, c32890, e33420, e33430,
+               e33450, e33460, e33465, e33470, _sep, _dclim, e32750, e32775,
                _earned):
 
-    # Part III ofdependent care benefits
-    if _cmp == 1 and MARS == 2:
+    # Part III of dependent care benefits
+    if f2441 != 0 and MARS == 2:
```

Comment:
`f2441`, Child Care Credit Qualified Individual, is not zero means that there're qualifying individual(s). Not sure why we are using `_cmp` in the first place.